app/gen-server/entity/Activation.ts

```diff
@@ -58,6 +58,8 @@ export class Activation extends BaseEntity {
         }
       }
 
+      this.prefs.checkForLatestVersion = props.prefs.checkForLatestVersion;
```

Comment:
Can you confirm that you can call updateFromProperties to set telemetry without resetting checkForLatestVersion?